pwa/app/components/tba/teamAvatar.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import { useState } from 'react';
+import { Media } from '~/api/v3';
+import { cn } from '~/lib/utils';
+
+export default function TeamAvatar({ media }: { media: Media }): JSX.Element {
+  const [colorClass, setColorClass] = useState('bg-blue-500');
+
+  return (
+    <img
+      alt="Team Avatar"
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      src={`data:image/png;base64, ${media.details['base64Image']}`}
+      className={cn(
+        'size-12 rounded inline mr-2 p-1 cursor-pointer',
+        colorClass,
+      )}
+      onClick={() => {
+        if (colorClass === 'bg-blue-500') {
```

Comment:
Can we use the official FIRST red/blue values that they specify for the audience display?

Reply:
Yeah, do you know where I can find them?

Reply:
https://github.com/the-blue-alliance/the-blue-alliance/pull/6020/files